vic/vic_run/include/vic_def.h

```diff
@@ -1131,6 +1131,7 @@ typedef struct {
     double aero_resist;           /**< Aerodynamic resistance (s/m) after stability correction */
     double density[MAX_LAKE_NODES]; /**< Lake water density profile (kg/m^3) */
     // Moisture fluxes
+    double influx                 /**< Volume of additional irrigation water amount into lake (m3) */
```

Comment:
If you want to keep this as a "permanent" variable in the `lake_var` struct, I recommend naming it in a more descriptive way - something like "lake_irrig" or "irrig_lake".

Reply:
(note: our convention in the UW-Hydro GitHub projects is to surround the names of variables, functions, and files, and any other pieces of code, with back-quotes (different from regular single quotes and from double quotes). This will make a gray box around the word, like I did for `lake_var`. See documentation on Markdown for more info on how to do this.

Reply:
OK, I found the reason the travis build failed. You need a semi-colon, as in `double influx;`  I think if you fix this, it will compile.